app/controllers/tickets_controller.rb

```diff
@@ -66,10 +80,39 @@ def find
   end
 
   def create_params
-    params.require(:data).permit(:title, :worker_id, :description, :state)
+    params.require(:data).permit(:title, :worker_id, :description)
   end
 
   def update_params
     params.require(:data).permit(:title, :description)
   end
+
+  def state(state_method, can_change_state)
```

Comment:
`can_change_state` is always true when this method is called according logic in controller.